interceptors_logger.go

```diff
@@ -142,7 +142,7 @@ func (s *loggingInterceptor) handle(
 	}
 
 	// Print the response data if it is set.
-	if rep != nil {
+	if !isResponseNil(method, rep) && rep != nil {
```

Comment:
@akutz when I didn't keep the `&& rep !- nil` portion here, I got crashes when trying to run the tests. The crashes were similar to what I was trying to fix -- reflect being called on an empty value. It was as if `isResponseNil` was missing entries in the map for detecting a nil response (but non-Nil interface).  I was unable to determine what was causing it.

Reply:
Hi Travis,

Yep. That’s because in some cases rep is actually nil. I’d prefer doing the more basic nil check in the isResponseNil func.